layouts/partials/frontpage-announcement.html

```diff
@@ -1,14 +1,29 @@
-{{ if .Page.Param "announcement" }}
-<section lang="en" id="fp-announcement" style="background-color:{{ .Page.Param "announcement_bg" }}">
-  <aside >
+{{ $announcementShown := false }}
```

Comment:
I considered refactoring this to be a single layout or partial that sets a different ID for the front page, or even making the HTML the same for both cases and using CSS to adjust styling. I decided that'd be best saved for a different PR; this one already has an potential impact on every page on the site(!) so let's keep it narrow.